### DISCUSSION TITLE
How to manage saved passwords and addresses

### DISCUSSION THREAD

**Discussion by naldobr** [MicrosoftEdge/WebView2Feedback#4904] [Q&A]
Recently, the WebView2 Runtime removed the Wallet settings page, where it was possible to configure and manage addresses and passwords saved by WebView2, such as viewing, editing and deleting. Now I don't know if it's possible to do this via any WebView2 API in my WPF/C# project. Can anyone help me?

**Comment by jls28** [comment]
If the wallet page is definitively blocked (seem true from runtime versions > 130.*),
an set of methods/functions to manage passwords will be 
very usefull.

**Reply by naldobr** [reply]
They had already warned that Wallet would be permanently removed. Without these methods to manage the records, the solution is to disable and recreate the resource manually :(
